config/develop/namespaced/glue-workflow.yaml

```diff
@@ -3,10 +3,11 @@ template:
 stack_name: "{{ stack_group_config.namespace }}-glue-workflow"
 dependencies:
   - develop/namespaced/glue-tables.yaml
+  - develop/namespaced/glue-job-S3ToJsonS3.yaml
```

Comment:
Can you explain why you added this?

Reply:
The glue workflow needs the S3 to JSON job name to create the S3 to JSON job trigger [here] (https://github.com/Sage-Bionetworks/recover/blob/main/templates/glue-workflow.j2#L69) so the S3 to JSON job stack needs to be created before this `glue-workflow` stack gets created. Before, deployment was failing because it was trying to launch the `glue-workflow` stack but it couldn't find the S3 to JSON job stack to get the name.